litellm/utils.py

```diff
@@ -2357,12 +2357,16 @@ async def wrapper_async(*args, **kwargs):
             print_verbose(
                 f"Async Wrapper: Completed Call, calling async_success_handler: {logging_obj.async_success_handler}"
             )
-            asyncio.create_task(
-                logging_obj.async_success_handler(result, start_time, end_time)
-            )
+            # asyncio.to_thread(
+            #     logging_obj.async_success_handler(result, start_time, end_time)
+            # )
             threading.Thread(
                 target=logging_obj.success_handler, args=(result, start_time, end_time)
             ).start()
+            threading.Thread(
```

Comment:
@ishaan-jaff how does switching create task for threads solve the issue? 

i'm also concerned about creating too many threads here, which would cause issues in high-traffic

Reply:
@krrishdholakia I update the conversation of this PR with notes